recipes/dftd3-python/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "dftd3-python" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/awvwgk/simple-dftd3/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 79b507a0d892f19222e1403f4c7ebb98ce9eb96b3b8d4821ba5631aa9ac756be
+
+build:
+  number: 0
+  skip: True  # [win or (osx and (python_impl == 'cpython' and py<38))]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - meson 0.55.*
+    - python
+    - cffi
+    - pkgconfig
+  host:
+    - python
+    - pip
+    - simple-dftd3
+    - libblas
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux and not aarch64]
+  run:
+    - python
+    - cffi
+    - numpy
+    - {{ pin_compatible('simple-dftd3', max_pin='x.x') }}
```

Comment:
You should instead rather add a `run_exports` to `simple-dftd3`

Reply:
Updated in upstream feedstock